applications/MeshingApplication/custom_processes/mmg/mmg_process.cpp

```diff
@@ -1277,6 +1285,7 @@ const Parameters MmgProcess<TMMGLibrary>::GetDefaultParameters() const
         {
             "isosurface_variable"              : "DISTANCE",
             "nonhistorical_variable"           : false,
+            "use_metric_field"                 : true,
```

Comment:
Should we use the new behaviour as default, or keep the isosurface discretization only (no adaptation) as default?

Reply:
I am in favour of not breaking anything, so keep current behaviour

Reply:
Ok thanks, now the default is `"use_metric_field" : false,`